### PR TITLE
Create now you need to tell us about limited partners page

### PR DESCRIPTION
**Pull request by HatiDev**
### JIRA link

https://companieshouse.atlassian.net/browse/LP-238

### Change description

Create now you need to tell us about limited partners page

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/presentation/controller/registration/PageType.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 enum RegistrationPageType {
   whichType = "which-type",
-  name = "name",
   email = "email",
   generalPartners = "general-partners",
+  limitedPartners = "limited-partners",
+  name = "name",
```
Comment (lduranteau) [inline src/presentation/controller/registration/PageType.ts:6]: "name" page is before "email" page

File: src/presentation/controller/registration/Routing.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const EMAIL_TEMPLATE = RegistrationPageType.email;
 export const GENERAL_PARTNERS_TEMPLATE = RegistrationPageType.generalPartners;
 
 export const NEXT_TEMPLATE = RegistrationPageType.next;
+export const LIMITED_PARTNERS_TEMPLATE = RegistrationPageType.limitedPartners;
```
Comment (lduranteau) [inline src/presentation/controller/registration/Routing.ts:17]: can you move it to line 15
should stay in order
the next page is always the last one for the moment

File: src/presentation/controller/registration/Routing.ts [hunk 3]
```diff
@@ -23,6 +24,8 @@ export const GENERAL_PARTNERS_URL = `${BASE_WITH_IDS_URL}/${GENERAL_PARTNERS_TEM
 
 export const NEXT_URL = `${BASE_WITH_IDS_URL}/${NEXT_TEMPLATE}`;
 
+export const LIMITED_PARTNERS_URL = `${BASE_WITH_IDS_URL}/${LIMITED_PARTNERS_TEMPLATE}`;
```
Comment (lduranteau) [inline src/presentation/controller/registration/Routing.ts:27]: same as above
URLs should be in order

File: src/presentation/controller/registration/Routing.ts [hunk 4]
```diff
@@ -37,6 +40,13 @@ const registrationRoutingName = {
   pageType: RegistrationPageType.name,
 };
 
+const registrationRoutingLimitedPartners = {
```
Comment (lduranteau) [inline src/presentation/controller/registration/Routing.ts:43]: same as above
order

File: src/presentation/controller/registration/Routing.ts [hunk 5]
```diff
@@ -64,6 +74,7 @@ const list = [
   registrationRoutingEmail,
   registrationRoutingGeneralPartners,
   registrationRoutingNext,
+  registrationRoutingLimitedPartners,
```
Comment (lduranteau) [inline src/presentation/controller/registration/Routing.ts:77]: same as above
order

File: src/routes/registration.ts [hunk 6]
```diff
@@ -58,6 +59,12 @@ export const registrationEndpoints = (
     authentication,
     dependencies.registrationController.getPageRouting()
   );
+
+  router.get(
```
Comment (lduranteau) [inline src/routes/registration.ts:63]: same as above
order

**Comment by lduranteau** [comment]
there are two code smells

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by markpit** [comment]
Needs a test to test the page

File: src/views/limited-partners.njk [hunk 7]
```diff
@@ -0,0 +1,19 @@
+{% extends "layout.njk" %}
+
+{% block beforeContent %}
```
Comment (lduranteau) [inline src/views/limited-partners.njk:3]: don't need this block
the backlink already includes by default

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/limited-partners.njk (hunk 7 above)
Comment (HatiDev) [reply src/views/limited-partners.njk]: block removed

File: src/routes/registration.ts (hunk 6 above)
Comment (HatiDev) [reply src/routes/registration.ts:63]: order changed

File: src/presentation/controller/registration/Routing.ts (hunk 5 above)
Comment (HatiDev) [reply src/presentation/controller/registration/Routing.ts:76]: order changed

File: src/presentation/controller/registration/Routing.ts (hunk 4 above)
Comment (HatiDev) [reply src/presentation/controller/registration/Routing.ts]: order changed

File: src/presentation/controller/registration/Routing.ts (hunk 3 above)
Comment (HatiDev) [reply src/presentation/controller/registration/Routing.ts:25]: order changed

File: src/presentation/controller/registration/Routing.ts (hunk 2 above)
Comment (HatiDev) [reply src/presentation/controller/registration/Routing.ts]: order changed

File: src/presentation/controller/registration/PageType.ts (hunk 1 above)
Comment (HatiDev) [reply src/presentation/controller/registration/PageType.ts]: order changed

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by lduranteau — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**